### ISSUE TITLE
BayesianHMSM fails for option init_hmsm

### ISSUE THREAD

**Issue by marscher** [markovmodel/PyEMMA#1189]
the estimate method copies over various attributes, but does not catch all attributes of importance and just crashes. @thempel you experienced this problem as well and you got a workaround for it, right?

Note that this option was simply not tested (and therefore broken). So it is deleted in the current devel branch. We can restore it as soon we got a working impl and test.

**Comment by thempel** [comment]
Yes, a functioning temporary workaround is to simply update `BayesianHMSM.__dict__` in the appropriate case of the `BayesianHMSM._estimate()` method.

```python
exclude_keys = ['stride', '_estimated']
self.__dict__.update(
{k: self.init_hmsm.__dict__[k] for k in self.init_hmsm.__dict__.keys() if k not in exclude_keys})

```
It would be better to whitelist, though, if we use this approach at all. Also one should discuss which arguments should be used in case a user delivers different choices for the init_hmm and the Bayesian one.

**Comment by marscher** [comment]
fixed by #1199
